### PR TITLE
Bump rdma-core to 34.1 and make it suitable for Prefix

### PR DESCRIPTION
**Pull request by bedroge**
Signed-off-by: Bob Dröge <b.e.droge@rug.nl>

### CODE REVIEW

**Comment by bedroge** [comment]
@amadio suggested that the following:
```
        if use python; then
                mycmakeargs+=( -DNO_PYVERBS=OFF -DPYTHON_EXECUTABLE="${PYTHON}" )
        else
                mycmakeargs+=( -DNO_PYVERBS=ON )
        fi
```
can perhaps be reduced to:
```
        mycmakeargs+=( -DNO_PYVERBS=$(usex python OFF ON) )
```
?

**Comment by amadio** [comment]
> @amadio suggested that the following:
> 
> ```
>         if use python; then
>                 mycmakeargs+=( -DNO_PYVERBS=OFF -DPYTHON_EXECUTABLE="${PYTHON}" )
>         else
>                 mycmakeargs+=( -DNO_PYVERBS=ON )
>         fi
> ```
> 
> can perhaps be reduced to:
> 
> ```
>         mycmakeargs+=( -DNO_PYVERBS=$(usex python OFF ON) )
> ```
> 
> ?

Yes, please do that, you can add it to the list above and remove this if statement completely.

**Comment by amadio** [comment]
Also, Gentoo has a convention for commit messages, so the message should be something like:
```sys-cluster/rdma-core: version bump, make it suitable for prefix```
You should also use `repoman commit` if you can, so that it adds a signoff line for you (or `git commit --signoff` at least).
(I can do these things upon merging, but it's good to get in the habit of following these, as it will speedup merging of future merge requests, and help catch errors).

**Comment by bedroge** [comment]
Ah, good to know, thanks! Will do that next time.

**Comment by Zlogene** [comment]
Hello, please fix commits headline.

File: sys-cluster/rdma-core/rdma-core-34.1.ebuild [hunk 1]
```diff
@@ -76,6 +76,7 @@ src_configure() {
 		-DENABLE_VALGRIND="$(usex valgrind)"
 		-DENABLE_RESOLVE_NEIGH="$(usex neigh)"
 		-DENABLE_STATIC="$(usex static-libs)"
+		-DNO_PYVERBS="$(usex python OFF ON)"
 	)
 
 	if use python; then
```
Comment (amadio) [inline sys-cluster/rdma-core/rdma-core-34.1.ebuild:82]: If you do that, you should remove this if statement, no?

Comment (bedroge) [reply sys-cluster/rdma-core/rdma-core-34.1.ebuild:82]: Oops, sorry, forgot to remove those other lines, apparently. I'll fix that...

**Comment by Zlogene** [comment]
@bedroge: few notes:

1. Please squash your python improvements into a single bump commit
2.  You [rpbably wanna try to update it to 35.0 (yes, delays on my end with merges, sorry)

**Comment by bedroge** [comment]
@Zlogene I hadn't noticed that 35.0 was already available, but I quickly tested it, and it seems to work with the same ebuild. Do you want me to close this PR and open a new one for 35.0?

**Comment by Zlogene** [comment]
@bedroge whatever most convinient for you :)

**Comment by bedroge** [comment]
@Zlogene: it took a bit longer due to vacation, but I just opened a new PR for rdma-core 36.0: #22092. So I'm closing this PR.

**Pull request closed without merging**